### DISCUSSION TITLE
Week timer

### DISCUSSION THREAD

**Discussion by almostlunatic** [jmcollin78/versatile_thermostat#237] [Ideas]
Would it be possible to add a week timer to switch on/off a device and the presets? Actually, the same functionality as all production heating thermostats.

**Comment by jmcollin78** [comment]
What is a week timer ?

**Comment by almostlunatic** [comment]
To be able to set on/off times for every day of the week, as I wrote. To switch the device and the 4 presets...

Here, it's at left on a Tuya thermostat:

<img width="333" alt="N7dZd5Qkrx" src="https://github.com/jmcollin78/versatile_thermostat/assets/23704588/a3be04b5-7b86-4d71-a5c5-03b989a7cba8">

**Reply by jmcollin78** [reply]
But this exists natively with the Scheduler of HA. Cf. Readme to have example.

**Reply by almostlunatic** [reply]
Correct! But what I meant is to give to my children the access only to the thermostat of their own rooms. They cannot adjust timings by HA.

**Reply by jmcollin78** [reply]
Ok but do you think that a specific implementation of a scheduler in Versatile Thermostat will help to give your children the access to their own room ? 

If you use the native HA Scheduler, you can have one scheduler per room. If your children have access to the VTherm they also can have access to the Scheduler.

I will not implement a scheduler in the VTherm which does the same as Scheduler but without many functionalities of the native HA Scheduler.

**Reply by almostlunatic** [reply]
Yeah, no problem! Done with HA Scheduler.
Tnx
